Documentation/Packaging/MRTK_Modularization.md

```diff
@@ -48,62 +21,29 @@ through the Mixed Reality Toolkit core component.
 
 ![Using the spatial awareness system via interfaces](../../Documentation/Images/Packaging/AccessingViaInterfaces.png)
 
-### Manual Componentization
-
-While automatic support for importing specific MRTK components is not available in RC1, developers can manually 
-opt-out of specific functionality.
-
-There are two workflows that developers can use to manually componentize MRTK v2:
-
-- [Uncheck features](#uncheck-features) during package import
-- [Delete feature folders](#delete-feature-folders)
-
-> If a feature is mistakenly omitted or deleted, the package can be re-imported to add the desired functionality.
-
-#### Uncheck Features
+### Minimizing MRTK import footprint
 
-When importing the Microsoft.MixedReality.Toolkit.Unity.Foundation-v2.0.0-RC1.unity package, developers have the 
-ability to limit which package contents get added to the project. 
+At this moment, the MRTK is imported as a single foundation package (ignoring for a moment the existence of
+the examples package, which is a completely optional package). It is possible to make this footprint smaller
+by manually cutting down on the files imported, though this is a highly manual process which doesn't have
+a well-defined guide.
 
-For projects where the required features and/or platforms are well known, unnecessary system services, providers 
-and features can be left unchecked to minimize the MRTK footprint.
-
-> NOTE: MRTK v2 **_requires_** the contents of the Assets\MixedRealityToolkit folder. 
-
-#### Delete Feature Folders
-
-After importing the Microsoft.MixedReality.Toolkit.Unity.Foundation-v2.0.0-RC1.unity package, the MRTK footprint can 
-be reduced by deleting unneeded folders from:
+It is possible to, during the import of the Foundation package, uncheck arbitrary items, though obviously this
```

Comment:
i'm not particularly fond of this paragraph. It reads a bit weird to me. Also we should avoid using "you" and "your" in the docs
My suggestion:
"It is possible to uncheck arbitrary items during the import of the Foundation package. However it's not recommended to do this at an early stage of development as it might break functionality. 
After having figured out the final feature set of the app, pruning unneeded providers and services can be done on the following folders:
"

Reply:
mhh the last sentence probably should be ... unneeded providers and services can be done in the following folders . sry not my native languag :D

Reply:
That's much better, changed to that!